pom.xml

```diff
@@ -279,6 +279,13 @@
         <artifactId>jchronic</artifactId>
         <version>0.2.6</version>
       </dependency>
+      <dependency>
+        <groupId>io.opentelemetry</groupId>
+        <artifactId>opentelemetry-bom-alpha</artifactId>
+        <version>1.6.0-alpha</version>
```

Comment:
Alpha ?

Reply:
The OTel community is stating this is alpha, even though the parts we're using (Tracing) is considered stable.